R/catchmentArea.R

```diff
@@ -594,28 +599,37 @@ calc_network_catchment <- function(sln,
                                  output = "epath")
     })
 
-    spaths <- unlist(lapply(spaths, function(x){x$epath}),recursive = FALSE)
-
-  }
+    spaths <-
+      unlist(lapply(spaths, function(x) {
+        x$epath
+      }), recursive = FALSE)
 
   }
 
-    spaths <- igraph::get.shortest.paths(sln@g, targetlayer,
-                                         which(sp::spDists(
-                                           x = as.matrix(data.frame(x=sln@g$x, y=sln@g$y)),
-                                           y = matrix(cbind(sln@g$x,sln@g$y)[targetlayer,],ncol=2),
-                                           longlat = longlat
-                                         ) <= maximpedance),
+  else {
+    spaths <- igraph::get.shortest.paths(sln@g,
```

Comment:
@mpadge any ideas if it will be possible to plug-in a [dodgr](https://github.com/mpadge/dodgr) equivalent here?

Richard, it seems dodgr's shortest path alg is substantially (~10 times) faster than igraph's.